Job `command`s that are too short cause errors

Job commands that cause the container to exit too quickly cause errors like this one:
```
[ERROR] signal=window changed error=API error (409): Container b404f88469de81b7d44d8393dc086866277a25e6bc5d0c6928e7f6dce1edba1a is not running [job: populate] sh -c 'cp /source/datafile /data; touch /data/newfile' Failed to set container's TTY window size
```
~~This causes the `init-named-volume` example/integration test to fail because the copy and touch commands run near-instantly.~~ (This might not be true actually, my test failure may be coming from a different cause, but the `ERROR` message still appears.) Brief testing shows that the minimum stable command length on my machine is 10 ms. 

If a solution to this error cannot be found, a `sleep 0.1s` should be added to the `populate` job's `command` in `init-named-volume`.

--- a/.gitignore
+++ b/.gitignore
@@ -7,3 +7,4 @@ docs/doctrees
 _results/
 .depsources/
 github.token
+dobi

--- a/examples/env-vars/test.sh
+++ b/examples/env-vars/test.sh
@@ -5,6 +5,12 @@
 # AUTHOR:
 set -eu -o pipefail
 
+cleanup() {
+    echo "running cleanup"
+    dobi autoclean
+}
+trap "cleanup" EXIT
+
 dobi dist
 
 echo "image tagged with version"

--- a/tasks/job/run.go
+++ b/tasks/job/run.go
@@ -400,7 +400,7 @@ func handleWinSizeChangeSignal(
 	err = client.ResizeContainerTTY(containerID, int(winsize.Height), int(winsize.Width))
 	if err != nil {
 		logger.WithError(err).
-			Error("Failed to set container's TTY window size")
+			Warning("Failed to set container's TTY window size.")
 	}
 }
 

